Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -1634,21 +1634,105 @@ static Ref<CSSValueList> valueListForAnimationOrTransitionProperty(CSSPropertyID
     return CSSValueList::createCommaSeparated(WTFMove(list));
 }
 
-static Ref<CSSValueList> animationShorthandValue(CSSPropertyID property, const AnimationList* animationList)
-{
-    CSSValueListBuilder parentList;
-    auto addAnimation = [&](Ref<Animation> animation) {
-        CSSValueListBuilder childList;
-        for (auto longhand : shorthandForProperty(property))
-            addValueForAnimationPropertyToList(childList, longhand, animation.ptr());
-        parentList.append(CSSValueList::createSpaceSeparated(WTFMove(childList)));
+static Ref<CSSValue> singleAnimationValue(const Animation& animation)
```

Comment:
Fwiw, you can leave `animationShorthandValue` untouched (for CSS Typed OM) and use ShorthandSerializer to serialize the computed style: https://searchfox.org/wubkat/rev/e0c29e7e482a4a452c92c394677358659c71eace/Source/WebCore/css/CSSComputedStyleDeclaration.cpp#120-130

But what you have now is fine too.